dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -216,10 +219,12 @@ class updatedns
                 }
                 break;
             case 'duckdns':
+	    case 'dnyv6':
+	    case 'dynv6-v6':
             case 'regfish':
             case 'regfish-v6':
                 if (!$dnsUser) {
-                    $this->_error(3);
+                    $this->_error(11);
```

Comment:
```suggestion
                    $this->_error(3);
```

almost there :) let's dial this new error back. the intention is good, but I don't think it adds clarity. can also remove the new error message below

Reply:
all right. So letd remove error handling back to original ....
will fixup with last commit.